setup.sh

```diff
@@ -49,11 +49,11 @@ install(){
 # Run tasks to build the project for distribution.
 build(){
   echo 'Building project...'
-  grunt clean
-  grunt
+  gulp clean
```

Comment:
Why? `setup.sh` should delegate logic to whatever build system is in use whenever possible.